spec/bolt/cli_spec.rb

```diff
@@ -974,6 +975,19 @@ def stub_config(config, file_content = nil)
             ]
           )
         end
+
+        it "plan with upper case task name errors" do
+          options = {
+            nodes: node_names,
+            mode: 'plan',
+            action: 'run',
+            object: 'sample::upper_case',
+            task_options: { 'nodes' => nodes.join(',') }
+          }
+
+          expect { cli.execute(options) }.to raise_error "Exiting because of an error in Puppet code"
```

Comment:
I get the following here:
```
............................................Error: Sample::Echo is not a valid task name  at /home/lucy/githubs/bolt/spec/fixtures/modules/sample/plans/upper_case.pp:3 on node ophelia.delivery.puppetlabs.net
{"kind":"bolt/cli-error","msg":"Exiting because of an error in Puppet code","details":{}}
..............................................................................................................................

Finished in 4.31 seconds (files took 0.46992 seconds to load)
205 examples, 0 failures
```

How do I catch that error, and not the 'Exiting because of an error' line? Should I be testing that in https://github.com/puppetlabs/puppet/pull/6474 ?